sample_gallery/nl2sql/README.MD

```diff
@@ -0,0 +1,110 @@
+	- Prompt used - [Validate SQL Prompt](./validate_sql_query.jinja2)
+  
+  *Note:* Prompts in this sample are specific to generate T-SQL complaint queries for [AdventureWorksLT](https://learn.microsoft.com/en-us/sql/samples/adventureworks-install-configure?view=sql-server-ver16&tabs=ssms) sample database hosted on Azure. However, you can modify the prompts as needed to work with other database systems.
+  
+## Getting Started
+
+These instructions will get you a copy of the sample up and running using PromptFlow for development and testing purposes.
+
+### Prerequisites
+
+ 1. The libraries required to run this project are listed in the `requirements.txt` file. To install them, use the following command: 
+
+```bash
+    pip  install  -r  requirements.txt
+```
+
+2. Deploy an Azure SQL Database with sample (AdventureWorksLT) - [Setup Guide)([Create a single database - Azure SQL Database | Microsoft Learn](https://learn.microsoft.com/en-us/azure/azure-sql/database/single-database-create-quickstart?view=azuresql&tabs=azure-portal)
+3. Install and setup the [ODBC Driver for Microsoft SQL Server (17)](https://learn.microsoft.com/en-us/sql/connect/odbc/download-odbc-driver-for-sql-server?view=sql-server-ver16#version-17) to connect to the Azure SQL Database using pyodbc
+	- (Local PromptFlow): Install the driver on the development machine
+	- (Azure AI Studio): The Automatic Runtime doesn't contain the ODBC Driver, so you need to create a custom runtime with promptflow, pyodbc, and the driver installed. [Customize Runtime](https://learn.microsoft.com/en-us/azure/ai-studio/how-to/create-manage-runtime)
+4. (Optional but highly recommended) Install and configure [Prompt flow for VS Code extension](https://marketplace.visualstudio.com/items?itemName=prompt-flow.prompt-flow) follow [Quick Start Guide](https://microsoft.github.io/promptflow/how-to-guides/quick-start.html).
+
+    💡 This extension is optional but highly recommended for flow development and debugging.
+
+### Connections Setup
+
+We need to setup 2 connections to run this sample. [Manage PromptFlow Connections](https://microsoft.github.io/promptflow/how-to-guides/manage-connections.html)
+
+ 1. Azure OpenAI connection: Create a connection to you Azure OpenAI endpoint with a GPT-4 Model deployment. 
+ 2. Custom Connection: Create a custom connection for the Azure SQL Database. Sample YAML
+
+```YML
+$schema: https://azuremlschemas.azureedge.net/promptflow/latest/CustomConnection.schema.json
+name: "to_replace_with_connection_name"
+type: custom
+configs:
+  Server_name: "tcp:<server-name>.database.windows.net,1433"
+  User_name: "<sql-login-name>"
+  Database_name: "AdventureWorksLT"
+secrets:
+# Don't replace the '<user-input>' placeholder. The application will prompt you to enter a value when it runs.
+  Password: "<user-input>"
+```
+
+Once the connections are created, modify the python and LLM nodes in the flow.dag.yaml with the appropriate connections names.
+
```

Comment:
Perhaps we should include 'for example' in this context? I believe the following content is only providing 2 examples. If users only modify these two, they will encounter an error. Seems that all nodes in this flow have connection names that need to be modified.